query/releasesearch.go

```diff
@@ -301,6 +297,30 @@ type ReleaseSearchRequest struct {
 	Highlight      bool
 }
 
+const (
+	Simple         = "!!s:"
+	SimpleExtended = "!!se:"
+	Sitewide       = "!!sw:"
+	Standard       = "!!st"
+)
+
+var TemplateNames = map[string]string{Simple: "s", SimpleExtended: "ss", Sitewide: "sw", Standard: "st"}
```

Comment:
not really - the pattern is that the api package uses the services of the query package, and I don't really want to break that by moving the code in to the api package. I could make the query package 'internal', but that would break the pattern in our repos in general

Reply:
Well maybe I should have read the comment better - yes I can reduce the visibility of the variable!